### PR TITLE
Floor

### PR DESCRIPTION
**Pull request by leichtaustin**
### Description

Adding floor() content for php section

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [ x] All writings are my own.
- [ x] My entry follows the Codecademy Docs style guide.
- [ x] My changes generate no new warnings.
- [ x] I have performed a self-review of my own writing and code.
- [ x] I have checked my entry and corrected any misspellings.
- [ x] I have made corresponding changes to the documentation if needed.
- [ x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@leichtaustin I did a first review and left a few comments.

**Comment by leichtaustin** [comment]
Resubmitted the changes to include the php code block around example.

File: content/php/concepts/math-functions/terms/floor/floor.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+## Syntax
+
+```pseudo
+$result = floor(n);
+```
+
+Where `n` is the number to be rounded down to the nearest integer.
+
+## Example
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/floor/floor.md:29]:
```suggestion
## Example

```
Add line break.

File: content/php/concepts/math-functions/terms/floor/floor.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  - 'Math Functions'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`floor()`** function returns a rounded number down to the nearest integer and if necessary, returns a result.
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/floor/floor.md:17]:
```suggestion
The **`floor()`** function returns a rounded number down to the nearest integer.
```

File: content/php/concepts/math-functions/terms/floor/floor.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
+Title: 'floor()'
+Description: 'Returns a rounded number down to the nearest integer and if necessary, returns a result.'
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/floor/floor.md:3]:
```suggestion
Description: 'Returns a rounded number down to the nearest integer.'
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@leichtaustin A few more comments.

**Review by SSwiniarski — APPROVED** [review]
@leichtaustin Looks good for a second review.

File: content/php/concepts/math-functions/terms/floor/floor.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  - 'Arithmetic'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`floor()`** function returns a rounded number down to the nearest integer.
+
+The result returns a value type of float.
```
Comment (yangc95) [inline content/php/concepts/math-functions/terms/floor/floor.md:19]:
```suggestion
The result returns a value type of `float`.
```

File: content/php/concepts/math-functions/terms/floor/floor.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+```
+
+Where `n` is the number to be rounded down to the nearest integer.
+
+## Example
+
+```php
+floor(3.4)
+```
+In this example, the result will be a float value of 3 as the function rounds down 3.4 to 3
```
Comment (yangc95) [inline content/php/concepts/math-functions/terms/floor/floor.md:34]:
```suggestion

In this example, the result will be a float value of 3 as the function rounds down 3.4 to 3
```

**Comment by yangc95** [comment]
🥳 @leichtaustin live link: https://www.codecademy.com/resources/docs/php/math-functions/floor

Screenshot 2023-05-26 at 2 45 06 PM

It looks as if you're anonymous. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.
To add your GitHub username to your Codecademy profile:
1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

Make sure to set your visibility to “everyone”:


2.) Then log out and log back into Codecademy using your GitHub login:

If the steps above do not work, it may be due to an email or local configuration issue.
In addition, we may reach out in the future if any adjustments to these entries are required.


cc: @SSwiniarski

**Pull request merged**